src/Microsoft.AspNetCore.Mvc.Core/Routing/ControllerLinkGeneratorExtensions.cs

```diff
@@ -146,11 +146,11 @@ public static class ControllerLinkGeneratorExtensions
         public static string GetUriByAction(
             this LinkGenerator generator,
             HttpContext httpContext,
-            string action = default,
-            string controller = default,
-            object values = default,
-            string scheme = default,
-            HostString? host = default,
+            string action,
+            string controller,
+            object values,
+            string scheme,
+            HostString host,
```

Comment:
Note that the other overload of this method (without HttpContext) has basically the same signature now.